### PR TITLE
Add Changelog Page

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

- Adds a changelog page

## WARNING: There's mocked data in here, do not merge before updating content!

### CODE REVIEW

**Comment by TGlide** [comment]
> The pagination is a bit off because when there is page 2 - I still start at page 1 :-)

that is intended, it's not **true** pagination, page n contains all results from page 1 to page n

File: src/routes/changelog/entry/[entry]/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<svelte:head>
+    <!-- Titles -->
+    <title>{seo.title + CHANGELOG_TITLE_SUFFIX}</title>
+    <meta property="og:title" content={seo.title} />
+    <meta name="twitter:title" content={seo.title} />
+    <!-- Desscription -->
+    <!-- <meta name="description" content={description} />
+  <meta property="og:description" content={description} />
+  <meta name="twitter:description" content={description} /> -->
+    <!-- Image -->
```
Comment (ArmanNik) [inline src/routes/changelog/entry/[entry]/+page.svelte:28]: leftover?

**Review by ArmanNik — APPROVED** [review]
Excluding the placeholder changelogs LGTM

**Comment by gewenyu99** [comment]
Screenshot 2023-12-20 at 1 56 28 PM
@TGlide I think there's missing space after lists and after headings.

File: src/routes/changelog/entry/[entry]/+page.svelte (hunk 1 above)
Comment (TGlide) [reply src/routes/changelog/entry/[entry]/+page.svelte]: Nope, fixed!

**Pull request merged**